pkg/ss/handler.go

```diff
@@ -9,13 +9,13 @@ func handleResponseElement(elem interface{}, response *interface{}, e *error) {
 	switch elem.(type) {
 	case int64:
 		*response = elem.(int64)
-		*e = nil
 	case string:
 		*response = elem.(string)
-		*e = nil
 	case error:
 		*e = elem.(error)
 		*response = (*e).Error()
+	case []byte:
```

Comment:
Why is this switch case required, you should be able to pass interface downstream. There is probably an error with your contract where you are returning bool instead of error.

Reply:
fixed this.